server/src/models/Vote.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const Model = require("./Model")
+
+class Vote extends Model {
+  static get tableName() {
+    return "votes"
+  }
+
+  static get relationMappings() {
+    const { Podcast, User } = require('./index.js')
+    return {
+      users: {
+        relation: Model.BelongsToOneRelation,
+        modelClass: User,
+        join: {
+          from: "votes.userId",
+          to: "users.id"
+        }
+      },
```

Comment:
this one works, but the naming is improper. `users` should be `user`, since a vote can only have one user